yarn.lock

```diff
@@ -5176,16 +5235,8 @@ which@^2.0.1:
   dependencies:
     isexe "^2.0.0"
 
-"wrap-ansi-cjs@npm:wrap-ansi@^7.0.0":
-  version "7.0.0"
-  resolved "https://registry.yarnpkg.com/wrap-ansi/-/wrap-ansi-7.0.0.tgz#67e145cff510a6a6984bdf1152911d69d2eb9e43"
-  integrity sha512-YVGIj2kamLSTxw6NsZjoBxfSwsn0ycdesmc4p+Q21c5zPuZ1pl+NfxVdxPtdHvmNVOQ6XSYG4AUtyt/Fi7D16Q==
-  dependencies:
-    ansi-styles "^4.0.0"
-    string-width "^4.1.0"
-    strip-ansi "^6.0.0"
-
-wrap-ansi@^7.0.0:
+"wrap-ansi-cjs@npm:wrap-ansi@^7.0.0", wrap-ansi@^7.0.0:
+  name wrap-ansi-cjs
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/wrap-ansi/-/wrap-ansi-7.0.0.tgz#67e145cff510a6a6984bdf1152911d69d2eb9e43"
   integrity sha512-YVGIj2kamLSTxw6NsZjoBxfSwsn0ycdesmc4p+Q21c5zPuZ1pl+NfxVdxPtdHvmNVOQ6XSYG4AUtyt/Fi7D16Q==
```

Comment:
Package name aliasing seems to be too much for Yarn 1. Running `yarn` produces a warning and then running Jest produces a related error.
```
[3/5] Fetching packages...
warning Pattern ["wrap-ansi@^7.0.0"] is trying to unpack in the same destination ".cache/yarn/v6/npm-wrap-ansi-cjs-7.0.0-67e145cff510a6a6984bdf1152911d69d2eb9e43-integrity/node_modules/wrap-ansi-cjs" as pattern ["wrap-ansi-cjs@npm:wrap-ansi@^7.0.0"]. This could result in non-deterministic behavior, skipping.
...

$ jest
Error [ERR_REQUIRE_ESM]: require() of ES Module lyra/node_modules/wrap-ansi/index.js from lyra/node_modules/cliui/build/index.cjs not supported.
```

The package aliasing comes from `@isaacs/cliui` which is a weird fork of `cliui`. Lyra depends on it via `eslint-config-next` which is an offical part of Next. Yarn 1 would maybe have surived this macabre insult of using package name aliasing if it weren't for us also depending on two incompatible versions of the aliased package. The weird fork itself depends on version 8 while the original `cliui`, depends on version 7. The original `cliui` is a dependency of Lyra via Jest.

You can produce the exact same predicament by just following Next's instructions for setting up Jest: https://nextjs.org/docs/app/building-your-application/testing/jest
That works with npm but will, if you are cursed with the same amount of fortune as Lyra, not work with Yarn 1.

Yarn 1 is a sinking ship and only accepts security fixes. Let's abandon it and just use npm. Since Yarn was created, all the features that motivated it have been added to npm. The only exception being parallel execution but npm has added other features that make it very fast anyway, see https://p.datadoghq.eu/sb/d2wdprp9uki7gfks-c562c42f4dfd0ade4885690fa719c818

Reply:
I'm open for whatever allows you as a team to move forward quickly and comfortably. If that means upgrading to modern yarn, or moving away from yarn entirely, doesn't really matter to me. I trust you, and whatever we learn from the experience can then (potentially) be migrated over to [app.zetkin.org](https://github.com/zetkin/app.zetkin.org).

Reply:
Ah lol I had an old commit lying around that upgrades this project to modern Yarn so I just pushed it directly into this branch. I don't really care though. I'll remove the commit from the branch again. Gimme a shout once you have a PR ready @WULCAN and I'll update this PR once it's merged!

Reply:
:loudspeaker: #108 @henrycatalinismith

Reply:
I am with moving to yarn 4. 
AFAIK yarn install is way faster than npm install cuz npm doesn't cache package locally.
I don't know what happend in the last 2 years but that was the biggest advantage when moving to from npm to yarn way faster install

Reply:
Mmm my enthusiasm for npm was mainly about the fact that adopting literally anything would get me a green build here and unblock the work of building Lyra. Happy to make space for a more slow paced evaluation of the options now that the urgency is addressed!!